pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix

```diff
@@ -23,7 +23,9 @@
 
 let
   is64Bit = stdenv.hostPlatform.parsed.cpu.bits == 64;
-  isMultiBuild  = multiPkgs != null && is64Bit;
+
+  # use of glibc_multi is only supported on x86_64-linux
```

Comment:
```suggestion
  # use of glibc_multi is only supported on x86_64-linux
```

Reply:
I think this is right, if the above line remains. Otherwise its:
```
unrelated code
# comment
related code
```